diff --git a/node_modules/apexcharts/src/apexcharts.js b/node_modules/apexcharts/src/apexcharts.js
index 1cf65d3..993df80 100644
--- a/node_modules/apexcharts/src/apexcharts.js
+++ b/node_modules/apexcharts/src/apexcharts.js
@@ -13,7 +13,7 @@ import YAxis from './modules/axes/YAxis'
 import InitCtxVariables from './modules/helpers/InitCtxVariables'
 import Destroy from './modules/helpers/Destroy'
 import { addResizeListener, removeResizeListener } from './utils/Resize'
-import apexCSS from './assets/apexcharts.css'
+import apexCSS from './assets/apexcharts.css?inline'
 
 /**
  *
